### PR TITLE
tests/integ: fix outdated tests

### PR DESCRIPTION
**Pull request by JadenSimon**
## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by JadenSimon** [comment]
Installing via pyenv didn't work. May be a SAM CLI issue relating to Python 3.6/3.7 as 3.8 works fine.

File: buildspec/linuxIntegrationTests.yml [hunk 1]
```diff
@@ -26,6 +26,10 @@ phases:
             - '>/dev/null pip3 install --upgrade aws-sam-cli'
             - '>/dev/null pip3 install --upgrade awscli'
             - '>/dev/null pip3 install pylint'
+            # SAM CLI is ran inside of a pyenv shim so we need to install the versions with pyenv
+            - '>/dev/null pyenv install 3.6.15'
```
Comment (justinmk3) [inline buildspec/linuxIntegrationTests.yml:30]: Why would this have changed? We've been doing `apt-get install python3.6 python3.7 python3.8` (line 23).

Comment (JadenSimon) [reply buildspec/linuxIntegrationTests.yml:30]: Aren't we on a different image now? That's my best guess.

Comment (justinmk3) [reply buildspec/linuxIntegrationTests.yml]: Yes it appears to correlate with the image change.

**Pull request merged**